rtengine/ipwavelet.cc

```diff
@@ -968,7 +926,7 @@ void ImProcFunctions::ip_wavelet(LabImage * lab, LabImage * dst, int kall, const
 
                     //printf("Levwava after: %d\n",levwava);
                     if (levwava > 0) {
-                        wavelet_decomposition* adecomp = new wavelet_decomposition(labco->data + datalen, labco->W, labco->H, levwava, 1, skip, max(1, wavNestedLevels), DaubLen);
+                        wavelet_decomposition* adecomp = new wavelet_decomposition(labco->data + datalen, labco->W, labco->H, levwava, 1, skip, rtengine::max(1, wavNestedLevels), DaubLen);
```

Comment:
Can't see the `delete`, but maybe a `const std::unique_ptr<wavelet_decomposition>` suits here.

Reply:
Done